akka-docs/src/main/paradox/scala/guide/tutorial_3.md

```diff
@@ -1,253 +1,180 @@
 
-@@@ note
+### Message Ordering
 
-We used the `expectNoMsg()` helper method from @scala[`TestProbe`] @java[`TestKit`]. This assertion waits until the defined time-limit
-and fails if it receives any messages during this period. If no messages are received during the waiting period the
-assertion passes. It is usually a good idea to keep these timeouts low (but not too low) because they add significant
-test execution time otherwise.
+In Akka, for a given pair of actors, messages sent directly from the first to the second will not be received out-of-order. The word directly emphasizes that this guarantee only applies when sending with the tell operator directly to the final destination, but not when employing mediators.
 
-@@@
+If:
 
-Scala
-:   @@snip [DeviceSpec.scala]($code$/scala/tutorial_3/DeviceSpec.scala) { #device-registration-tests }
+ * Actor `A1` sends messages `M1`, `M2`, `M3` to `A2`.
+ * Actor `A3` sends messages `M4`, `M5`, `M6` to `A2`.
 
-Java
-:   @@snip [DeviceTest.java]($code$/java/jdocs/tutorial_3/DeviceTest.java) { #device-registration-tests }
+This means that, for Akka messages:
 
-## Device Group
+ * If `M1` is delivered it must be delivered before `M2` and `M3`.
+ * If `M2` is delivered it must be delivered before `M3`.
+ * If `M4` is delivered it must be delivered before `M5` and `M6`.
+ * If `M5` is delivered it must be delivered before `M6`.
+ * `A2` can see messages from `A1` interleaved with messages from `A3`.
+ * Since there is no guaranteed delivery, any of the messages may be dropped, i.e. not arrive at `A2`.
 
-We are done with the registration support at the device level, now we have to implement it at the group level. A group
-has more work to do when it comes to registrations. It must either forward the request to an existing child, or it
-should create one. To be able to look up child actors by their device IDs we will use a @scala[`Map[String, ActorRef]`] @java[`Map<String, ActorRef>`].
+For the full details on delivery guarantees please refer to the [reference page](http://doc.akka.io/docs/akka/current/general/message-delivery-reliability.html).
 
-We also want to keep the original sender of the request so that our device actor can reply directly. This is possible
-by using `forward` instead of the @scala[`!`] @java[`tell`] operator. The only difference between the two is that `forward` keeps the original
-sender while @scala[`!`] @java[`tell`] always sets the sender to be the current actor. Just like with our device actor, we ensure that we don't
-respond to wrong group IDs:
+Reviewers: I wasn't sure if the "This means that" list of bullets *is* the ordering that Akka provides? I've edited to make it read that way. And, I think a diagram here would be really helpful, as well as a summary of the benefits (again).
```

Comment:
Perhaps a sequence diagram showing messages from A1 can cross messages from A2 but not each other?